### ISSUE TITLE
issue-summary should include non-github issues

### ISSUE THREAD

**Issue by jyasskin** [w3c/respec#3942]
## Important info

- ReSpec version: 28.1.0
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

**What happened (e.g., it crashed)?:**

I have some `<aside class="issue">` in the document without a `data-number` attribute, meaning they're not filed in Github. The `<section id=issue-summary>` section omits these issues entirely.

**Expected behavior (e.g., it shouldn't crash):**

Either give an error for github-less issues or (preferred) include them in the summary.

**Comment by marcoscaceres** [comment]
Sorry for the delay... still catching up post vacation. Having a look to see how I can get them to show up.

**Comment by jyasskin** [comment]
#3972 is close, but issues that don't have a `title` attribute are missing their newlines: https://w3ctag.github.io/privacy-principles/#issue-summary 
![Issue links without newlines](https://user-images.githubusercontent.com/83420/151420910-39d4d63c-bd7a-4e03-a9a8-a177b4d27b2b.png)


Adding `title` attributes is a bit of a pain because for many issues, that's the whole content of the issue, but the attributes can't include embedded links, like for https://w3ctag.github.io/privacy-principles/#issue-container-generatedID-0 whose content is

```html
<aside class="issue">Broaden this from [=processing=] to general interactions with systems.</aside>
```

Maybe it'd make sense to use the first paragraph as the title for issues without a title?

**Comment by marcoscaceres** [comment]
Non-GitHub issues (without `data-number`) are already included in the issue summary. This works because the `issueList.append()` call is inside the `if (isIssue)` block but outside the `data-number` check. Verified by the existing test at line 26 of issues-notes-spec.js which uses `<aside class="issue" title="An issue">` without `data-number` and the summary asserts 4 items (line 44).

**Comment by jyasskin** [comment]
@marcoscaceres I haven't reproduced this recently, but your AI summary of why this issue is fixed doesn't mention my [comment in this thread](https://github.com/speced/respec/issues/3942#issuecomment-1023520166) that there's still a problem with title-less issues. Does your prompt include an instruction to go through the comment thread to look for any extra test cases to add, or follow-up bugs to open?

**Comment by marcoscaceres** [comment]
Reopening. Jeffrey correctly points out that title-less issues still have a rendering problem (missing newlines in the summary). The original closure only verified that non-GitHub issues are included, but didn't check the title-less case from the comment thread. Will investigate.
